src/plugins/condition/Condition.js

```diff
@@ -163,6 +163,7 @@ export default class ConditionClass extends EventEmitter {
         if (found) {
             let criterion = found.item;
             criterion.destroy();
+            // TODO this is passing the wrong args
```

Comment:
Can you create issues for these TODOs?

Reply:
@shefalijoshi , do you mind creating issues or you remove these bits of code directly in the PR if they are not needed. I didn't want to change anything I wasn't directly working on. I just noticed these bits of unconnected code but don't know if they have intended purposes or not.

Reply:
You added this comment for the issue, so I don't know what the issue is to create one :)
You'll have to do this one.